docs/website/versioned_docs/version-3.0.0/overview/configure.md

```diff
@@ -144,21 +144,4 @@ Successfully deleted registry
 
 You can use the `--force` (or `-f`) flag to force the deletion of the registry without confirmation.
 
-### Updating a registry
```

Comment:
Wondering if we should not keep this section, and just mention that users need to delete the registry and add it again... What do you think?